syntax = "proto3";

option optimize_for = LITE_RUNTIME;

message boneContainer_proto
{
	optional sint32 index = 1;
	optional sint32 parentIndex = 2;
	optional bytes nodeName = 3;
}